src/disclosure-manager/DisclosureManagerDelegate.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
```

Comment:
This is one-to-one with terra-disclosure-manager's DisclosureManagerDelegate. Do we need to reimplement or can we reuse the existing Delegate?

Reply:
Since we will be deprecating the existing packages, this is getting moved over wholesale.